sandbox/index.html.example

```diff
@@ -10,12 +10,6 @@
   <!-- Load the source files -->
   <link href="../build/temp/video-js.css" rel="stylesheet" type="text/css">
   <script src="../build/temp/video.js"></script>
-
-  <!-- Set the location of the flash SWF -->
```

Comment:
what about just adding the flash tech to this sandbox? That way if you're fiddling with this file and then run it on IE8, it won't just stop working.

Having a flash example which prefers flash as the tech order sounds good either way.